api/user/schema.py

```diff
@@ -52,15 +50,11 @@ def mutate(self, info, user_id, **kwargs):
         query_user = User.get_query(info)
         exact_query_user = query_user.filter(
             UserModel.id == user_id).first()
```

Comment:
Consider using emails to search for users, emails will be unique across Andela and better to use than id which might change from one Andela app to another